app/styles/components/addon-tabs.css

```diff
@@ -0,0 +1,58 @@
+.addon-tabs {
+  width: 460px;
```

Comment:
Can you create an issue to make this relative units please.

Reply:
It turns out that this wasn't needed any more, it works perfectly with `width: 100%` 👍